modules/luabackend/dnssec.cc

```diff
@@ -363,19 +363,16 @@ bool LUABackend::addDomainKey(const DNSName& name, const KeyData& key, int64_t&
     return ok >= 0;
 }
 
-bool LUABackend::getDomainKeys(const DNSName& name, unsigned int kind, std::vector<KeyData>& keys) {
-    //what is kind used for?
-
+bool LUABackend::getDomainKeys(const DNSName& name, std::vector<KeyData>& keys) {
     if(f_lua_getdomainkeys == 0) 
 	return false;
 
     if(logging)
-	L << Logger::Info << backend_name << "(getDomainKeys) BEGIN name: '" << name << "' kind: '" << kind << endl;
+	L << Logger::Info << backend_name << "(getDomainKeys) BEGIN name: '" << name << endl;
 
     lua_rawgeti(lua, LUA_REGISTRYINDEX, f_lua_getdomainkeys);
 
     lua_pushstring(lua, name.toString().c_str());
-    lua_pushinteger(lua, kind);
 
     if(lua_pcall(lua, 2, 1, f_lua_exec_error) != 0) {
```

Comment:
Also remove 1 argument here and update the function in https://github.com/PowerDNS/pdns/blob/master/modules/luabackend/test2/pdns-luabackend.lua